webclient/index.html

```diff
@@ -125,6 +139,8 @@
                 alert('Whoops. The cached files on your machine are out of sync with each other. That\'s our bad. Please hard-refresh the page?');
             }
         </script>
+<flip-card id="mainFlipCard">
```

Comment:
Here are some thoughts of mine on our user experience:

There are 2 types of users: the **order creater** inputs order details, decides a tip, and shares the results. The **order participant** simply checks the summary table to see how things add up.

How I envision the UX for an **order participant**:

- Clicks a link to the website and sees the order summary. Nothing else.

How I envision the UX for an **order creator**:

- Enters order details. Upon clicking the "split" button, the summary table becomes visible and the inputs are dismissed, except, _maybe_, for the tip field. Adjusting the tip field results in live updates to the summary table.